### DISCUSSION TITLE
ScriptReference/MonoBehaviour.OnRenderObject

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#18] [Notes]
# ScriptReference/MonoBehaviour.OnRenderObject

https://docs.unity3d.com/ScriptReference/MonoBehaviour.OnRenderObject.html

**Comment by vertxxyz** [comment]
This is not called in Edit Mode without the [`ExecuteInEditMode`](https://docs.unity3d.com/ScriptReference/ExecuteInEditMode.html) attribute.
